auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -525,4 +533,9 @@ export default defineComponent({
   }
 }
 
+.future-effective-icon {
+  color: #4169E1 !important;
+  margin-left: 4px;
```

Comment:
I recommend using a class of "ml-1" for alignment.

Also, can you use a color of `#F8661A` ? If that's acceptable then I think you can specify it right in the template (color="xxx"), so you don't need this CSS.

Reply:
Updated